lib/websql/WebSQLDatabase.js

```diff
@@ -127,4 +185,45 @@ WebSQLDatabase.prototype.readTransaction = function (txnCallback, errorCallback,
   this._createTransaction(true, txnCallback, errorCallback, successCallback);
 };
 
+WebSQLDatabase.prototype.changeVersion = 
+  function (oldVersion, newVersion, txnCallback, errorCallback, successCallback) {
+  var self = this;
+  /* istanbul ignore next */
+  errorCallback = errorCallback || noop;
+  /* istanbul ignore next */
+  successCallback = successCallback || noop;
+  self.transaction(function (tx) {
+    tx.executeSql(getVersion.sql, getVersion.args, function (tx, res) {
+      // Check existing verion
+      /* istanbul ignore next */
+      if (res.rows.length === 1) {
+        var currentVersion = res.rows.item(0).value;
+        if (oldVersion !== currentVersion) {
+          return errorCallback(tx, { 
+            code: 2,
+            message: 'current version of the database and `oldVersion` argument do not match'});
+        }
```

Comment:
Ditto about a real `Error` object here. [It's not clearly specified](https://www.w3.org/TR/webdatabase/#dom-database-changeversion), but it seems implied to me that this should be an `InvalidStateError` too.

> If the database version provided is not the empty string, and there is already a database with the given name from the origin origin, but **the database has a different version than the version provided**, then throw an `INVALID_STATE_ERR` exception